include/ginkgo/core/matrix/csr.hpp

```diff
@@ -129,8 +167,15 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         std::string name_;
     };
 
+    /**
+     * classical is a strategy_type which uses the same number of threads on
+     * each row.
+     */
     class classical : public strategy_type {
     public:
+        /**
+         * Creates a classical strategy.
+         */
         classical() : strategy_type("classical") {}
 
         void process(const Array<index_type> &mtx_row_ptrs,
```

Comment:
I think this and `clac_size` can be marked `override` here and for the other strategies below for clarity.